pages/post.js

```diff
@@ -3,26 +3,38 @@ import fetch from 'node-fetch'
 import Layout from '../components/Layout'
 import PostCard from '../components/PostCard'
 import CommentInput from '../components/CommentInput'
+import Comment from '../components/Comment'
 
 const Image = styled.img`
   width: 100%;
   height: 260px;
   object-fit: cover;
 `
 
+const Title = styled.h2`
+  margin: 40px 60px;
+  font-size: 28px;
+  font-family: ${(props) => props.theme.fontFamilyTitle};
+`
+const testComment = "Comentario de Prueba - Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley"
+
 function PostPage({ post = {} }) {
   const { media = {} } = post
   const { url: imgSrc = '', title: imgTitle = '' } = media[0]
   return (
     <Layout>
       <Image src={imgSrc} alt={imgTitle} />
       <PostCard post={post} />
+      <Title>Comments</Title>
       <CommentInput
         placeholder="Add your comment"
-        rows={1}
+        rows={2}
         handleClick={null}
         buttonText="Comment"
       />
+      {
```

Comment:
Why doesn't We import the <CommentsSection /> component instead of calling the component in here